### PR TITLE
Move Documentation from Wiki to GitHub

### PR DESCRIPTION
**Pull request by oleg-nenashev**
This pull request finishes the migration of docs to GitHub. It will help us to write new documentation for GSoC 2019

Browse link: https://github.com/jenkinsci/role-strategy-plugin/tree/docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,31 +6,58 @@ Role Strategy plugin
 [![GitHub release](https://img.shields.io/github/release/jenkinsci/role-strategy-plugin.svg?label=release)](https://github.com/jenkinsci/role-strategy-plugin/releases/latest)
 [![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/role-strategy.svg?color=blue)](https://plugins.jenkins.io/role-strategy)
 
-About this plugin
------------------
-The Role Strategy plugin is meant to be used from [Jenkins][2] to add a new role-based mechanism to manager users' permissions. Please take a look at [Jenkins' wiki][3] to get detailed information.
+## About this plugin
+
+The Role Strategy plugin is meant to be used from [Jenkins](https://jenkins.io) to add a new role-based mechanism to manager users' permissions.
```
Comment (AbhyudayaSharma) [inline README.md]:
```suggestion
The Role Strategy plugin is meant to be used from [Jenkins](https://jenkins.io) to add a new role-based mechanism to manage users' permissions. 
```

**Comment by oleg-nenashev** [comment]
CI is just overloaded by my pushes, there is no real checks which would verify the documentation

**Comment by oleg-nenashev** [comment]
poking the build

**Comment by oleg-nenashev** [comment]
I am confident that there is no regression in the production code caused by this commit. I am not sure what;s up with CI, but will merge it  and fix any issues in follow-ups

**Pull request merged**